services/admin/emoji/add.go

```diff
@@ -6,11 +6,19 @@ import (
 
 // AddRequest represents an Add request.
 type AddRequest struct {
+	Name   string `json:"name"`
 	FileID string `json:"fileId"`
 }
 
 // Validate the request.
 func (r AddRequest) Validate() error {
+	if r.Name == "" {
+		return core.RequestValidationError{
+			Request: r,
+			Message: core.UndefinedRequiredField,
+			Field:   "Name",
+		}
+	}
```

Comment:
For the linting issue:
> if statements should only be cuddled with assignments (wsl)

```suggestion
	}

```

Reply:
I am sorry. I'll fix it right away...!